Detectors/TRD/base/test/testDigit.cxx

```diff
@@ -0,0 +1,173 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file testTRDGeometry.cxx
+/// \brief This task tests the Geometry
+/// \author Sean Murray, murrays@cern.ch
+
+#define BOOST_TEST_MODULE Test TRD_Digit
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+#include <boost/test/unit_test.hpp>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include "TRDBase/Digit.h"
+#include <iostream>
+
+namespace o2
+{
+namespace trd
+{
+
+/// \brief Test the Geometry class
+//
+///
+//
+
+void testdigitpadrow(Digit& test, int det, int row, int pad)
```

Comment:
Can we rename this function to testDigitDetRowPad()?

Reply:
ja done